src/components/PeopleFilters.tsx

```diff
@@ -1,12 +1,60 @@
-export const PeopleFilters = () => {
+import classNames from 'classnames';
+import React from 'react';
+import { useSearchParams } from 'react-router-dom';
+import { SearchLink } from './SearchLink';
+
+const centuriesForButtons = [
```

Comment:
Nice!) but since it's always a static values you could just use an array like `[17,18,19,20]`